lib/travis/services/find_admin.rb

```diff
@@ -10,11 +10,11 @@ class FindAdmin < Base
       include Travis::Logging
 
       register :find_admin
+      NUM_CANDIDATES = 15
```

Comment:
@aakritigupta yep, that's right. i think it's better to keep re-trying for a little longer than just abort. since we do unset the respective permission inline it will sort of fix the issue and the time spent isn't wasted. next time a request comes in for the same repo it won't retry the wrong user again.